src/Extensions/AzureBlobStorage/extension.json

```diff
@@ -1,10 +1,4 @@
 {
-    "Id": "Microsoft.Diagnostics.Monitoring.AzureStorage",
```

Comment:
Did we want to leave version? For example, if there are multiple releases of a private egress extension, it might be worthwhile to know the version for troubleshooting purposes.

We could also add tags (similar to nuget package tags, but perhaps kv pairs) for arbitrary data that could be added to the extension manifest.

Reply:
I see the version as being fairly superfluous at the moment. Nothing uses it and the extension version is the same exact version as dotnet-monitor for the time being. We can add it back if we use it for something (logging, extension enumeration for debugging purposes through some configuration validation or a dedicated `dotnet-monitor extensions list` command, etc). I'll add this to the backlog though.